lib/runners/javascript.js

```diff
@@ -43,7 +43,6 @@ module.exports.run = function run(opts, cb) {
                         .replace(/( Object. )?[\(]?\[eval\][-:\w\d\)]* at/g, '')
                         .replace(/Module._compile.*/g, '')
                         .replace('Object.Test.handleError ', '')
-                        .replace('  ', ' ')
```

Comment:
Nice catch! :+1:
This should fix these:

- https://github.com/Codewars/codewars.com/issues/267
- https://github.com/Codewars/codewars.com/issues/499
- https://github.com/Codewars/codewars.com/issues/631